qiskit/quantum_info/__init__.py

```diff
@@ -104,7 +104,7 @@
    euler_angles_1q
    two_qubit_cnot_decompose
    TwoQubitBasisDecomposer
-
+   OneQubitEulerDecomposer
```

Comment:
I think this won't work unless `OneQubitEulerDecomposer` is imported here.

Reply:
Whoops forgot to add import